pkgs/by-name/bi/bink-player/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoPatchelfHook,
+  p7zip,
+  libGL,
+  libX11,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bink-player";
+  version = "2025.05";
+
+  src = fetchurl {
+    url = "https://www.radgametools.com/down/Bink/BinkLinuxPlayer.7z";
+    hash = "sha256-A3IDQtdYlIcU2U8uieQI6xe1SvW4BqH+5ZwPYJxr83M=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    p7zip
+  ];
+
+  buildInputs = [
+    stdenv.cc.cc.lib
+    libGL
+    libX11
+  ];
+
+  unpackPhase = ''
+    7z x $src
+  '';
```

```suggestion
  unpackPhase = ''
    runHook preUnpack

    7z x $src

    runHook postUnpack
  '';
```

Also, this should be right below `src =`.

---

Thank you!